source/scenes/menu.js

```diff
@@ -19,44 +20,74 @@ export class MenuScene extends Phaser.Scene{
     }
     create(){
         // create the menu screen
-
         this.add.image(0, 0, 'menu-bg').setOrigin(0,0).setDepth(0);
         this.add.image(400, 300, 'menu').setDepth(1);
         this.add.image(400, 125, 'rsc-logo').setDepth(1);
 
-        // Button hover sprites
-
-
-        // Button format
-/*
-        let buttonName = this.add.image(locationX, locationY, "button-name").setDepth(1);
-        buttonName.setInteractive();
-        buttonName.on("pointerover", ()=>{
-             stuff on hover 
-        }
-        buttonName.on("pointerover", ()=>{
-             stuff on hover 
-        
-        }
-        buttonName.on("pointerout", ()=>{
-             stuff on not hover 
-        }
-        buttonName.on("pointerup", ()=>{
-             stuff on click 
-        }
-*/
-        // Buttons
+        // Play
         let playButton = this.add.image(485, 321, "play-button").setDepth(1);
         playButton.setInteractive();
-        
-
         playButton.on("pointerup", ()=>{
-            this.scene.start(CONSTANTS.SCENES.CC); 
-            console.log("Going to Character Creation");
+            if (!this.levelData.characterClass) {
+                this.scene.start(CONSTANTS.SCENES.CC); 
+                console.log("Going to Character Creation");
+            }
+            else {
+                this.scene.start(CONSTANTS.SCENES.LUMBRIDGE, this.levelData); 
+                console.log("Going to Lumbridge");
+            }
         })
 
+        // Settings
         let settingsButton = this.add.image(319, 321, "settings-button").setDepth(1);
 
+        // Pull in previous data
+        this.getCookies();
+    }
+    getCookies(){
+        let cookieArray = ["gold", "characterClass", "enemiesKilled", "timesClicked", "damageByClicking"];
+
+        let decodedCookies = decodeURIComponent(document.cookie).split(';');
+        for (let i = 0; i < decodedCookies.length; i++) {
+            let cookie = decodedCookies[i];
+
+            // Let level know we have cookies
+            this.levelData.hasCookies = true;
+
+            // Remove starting whitespace
+            while (cookie[0] == ' ') {
+                cookie = cookie.substring(1);
+            }
+
+            // Check for all stored cookies
+            for (let k = 0; k < cookieArray.length; k++) {
```

Comment:
find a way to loop through and set these easily and get rid of the for loop you already have

Reply:
Okay